packages/jose/jose.0.1.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "JOSE implementation for OCaml and ReasonML"
+description:
+  "JavaScript Object Signing and Encryption built ontop of pure OCaml libs"
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "MIT"
+homepage: "https://ulrikstrid.github.io/reason-jose"
+doc: "https://ulrikstrid.github.io/reason-jose"
+bug-reports: "https://github.com/ulrikstrid/reason-jose/issues"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "base64"
+  "dune" {>= "1.11" & < "2.0"}
+  "nocrypto"
+  "x509" {>= "0.8.0"}
+  "astring"
+  "yojson"
+  "zarith"
+  "containers" {with-test}
+  "bisect_ppx" {with-test}
+  "reason" {with-test}
+  "rely" {with-test}
+  "lwt" {with-test}
+]
+dev-repo: "git+https://github.com/ulrikstrid/reason-jose.git"
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@doc" {with-doc}
+  ]
+  ["dune" "exec" "test/test_exe/RunTests.exe"] {with-test}
```

Comment:
```suggestion
  ["dune" "exec" "-p" name "test/test_exe/RunTests.exe"] {with-test}
```
In releases, dune should always be called with `-p name` or `--profile=release` as the default (dev) profile might turn new warnings into errors in the future.